docs/quickstart.rst

```diff
@@ -46,67 +46,31 @@ it as a user package or in the virtualenv we define below.
 
 .. code-block:: shell
 
-  # Install cookiecutter locally or upgrade it if already installed
-  $ pip install --user --upgrade cookiecutter
-  # OR, once you have created a virtualenv per the steps below, install it
-  (my-repository-venv)$ pip install --upgrade cookiecutter
-
-.. note::
-
-  If you install Cookiecutter in the virtualenv, you will need to activate the
-  virtualenv to be able to use `cookiecutter` on the command-line.
-
-
-We can now begin. First, let's create a `virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_
-using `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/install.html>`_
-in order to sandbox our Python environment for development:
-
-.. code-block:: shell
-
-  $ mkvirtualenv my-repository-venv
+  # Install cookiecutter if it is not already installed
+  $ sudo apt-get install cookiecutter
```

Comment:
Probably this can be improved, it was changed as we're not showing `pip install` or any virtualenv